examples/csv2argus.py

```diff
@@ -0,0 +1,261 @@
+                continue
+            # Append rows to data
+            data.append(dict(zip(cols, row)))
+
+        if not opts.quiet:
+            logging.info("Total result count: %s", len(data))
+
+        # dictionary var to hold metrics
+        m_dict = {}
+        # for loop to populate m_dict
+        for row in data:
+            # abort without timestamp
+            try:
+                ts = row[opts.timestampcolumn] and to_gmt_epoch(row[opts.timestampcolumn])
+            except KeyError:
+                logging.error("Error: Timestamp not found: %s", row)
+                return None
+
+            # create final scope, substitute keys into the scope
+            rowScope = opts.argusscope
+
+            # abort without arguskeys
+            if keyNames:
+                for keyName in keyNames:
+                    logging.debug("Subbing keyName " + keyName + " with value " + str(row[keyName]))
+                    try:
+                        rowScope = rowScope.replace("{" + keyName + "}", row[keyName])
+                        logging.debug("New rowScope = " + rowScope)
+                    except KeyError:
+                        logging.error("Error: Specified arguskeys not found: %s", row)
+                        return None
+
+            # create tags
+            tag_dict = {}
+            if tagNames:
+                for tagName in tagNames:
+                    logging.debug("Setting tag pair for name: " + tagName + " and value: " + str(row[tagName]))
+                    # abort without argustags
+                    try:
+                        tag_dict[tagName] = row[tagName]
+                    except KeyError:
+                        logging.error("Error: Specified tags not found: %s", row)
+                        return None
+
+            # create metrics and datapoints
+            for col in metricNames:
+
+                # abort without argusmetrics
+                try:
+                    val = row[col]
+                except KeyError:
+                    # Key is not present
+                    logging.error("Error: Specified metrics not found: %s", row)
+                    return None
+
+                # abort for non-numeric metrics
+                try:
+                    float(val)
+                except:
+                    logging.error("Error: Non-numeric metric found: %s", row)
+                    return None
+
+                # cast str to number
+                if "." in val:
+                    val = float(val)
+                else:
+                    val = int(val)
+
+                # add a Metric object to m_dict if it doesn't already exist [namespace]:scope:metric{tags}
+                if opts.argusnamespace:
+                    metric = Metric(scope=rowScope, metric=col, tags=tag_dict, namespace=opts.argusnamespace)
+                else:
+                    metric = Metric(scope=rowScope, metric=col, tags=tag_dict)
+
+                metric_key = str(metric)
```

Comment:
I see you are using the full metric specification as the key instead of just the column name. May I know the reason? Do we need to update `splunk2argus.py` as well for the same reason?